src/client/pages/CountryHome/FraHome/hooks/useSections.tsx

```diff
@@ -8,7 +8,7 @@ import { Users } from 'meta/user'
 import { useAssessment, useCycle } from 'client/store/assessment'
 import { useUser } from 'client/store/user'
 import { useCountryRouteParams } from 'client/hooks/useRouteParams'
-import Collaborators from 'client/pages/CountryHome/FraHome/Collaborators'
+import Collaborators from 'client/pages/CountryHome/Collaborators'
```

Comment:
Collaborators page moved one level higher as it is shared by both PanEuropean and FRA Assessments